spec.bs

```diff
@@ -1440,8 +1440,22 @@ To <dfn export>generate a [=/pattern string=]</dfn> from a given [=/part list=]
       <ul>
         <li>|part|'s [=part/suffix=] is not the empty string.</li>
         <li>|part|'s [=part/prefix=] is not the empty string and is not |options|'s [=options/prefix code point=].</li>
-        <li>|custom name| is true, |part|'s [=part/type=] is "<a for=part/type>`segment-wildcard`</a>", |part|'s [=part/modifier=] is "<a for=part/modifier>`none`</a>", |next part| is not null, |next part|'s [=part/type=] is not "<a for=part/type>`fixed-text`</a>", |next part|'s [=part/prefix=] is the empty string, |next part|'s [=part/suffix=] is the empty string, and |next part|'s [=part/name=][0] is an [=ASCII digit=].
       </ul>
+    1. If all of the following are true:
```

Comment:
Preexisting issue found while reviewing the output: line 1428 should be `[=list/size=]` not `[=list/size]`